### PR TITLE
Make sudo removal

### PR DESCRIPTION
**Pull request by sylvinite**
# Description
- Removed sudo from make (no longer a req w a recent apptainer)
- Updated the config f NGP w some necessities
- Made the README prettier

## Primary function of PR
- [X] Hotfix
- [ ] Minor functionality improvement
- [ ] Major functionality improvement / New type of analysis
- [ ] Backward-breaking functionality

# Testing
ReadMe simple selftest

# Sign-offs
- [ ] Code reviewed by @octocat
- [x] Code tested by @sylvinite

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,67 +14,85 @@ Jasen has been developed for a small set of microbiota (primarily MRSA), but wil
...
-git clone --recurse-submodules --single-branch --branch master  git@github.com:genomic-medicine-sweden/jasen.git && cd jasen
+nextflow run main.nf -profile staphylococcus_aureus -config configs/nextflow.base.config --csv assets/test_data/samplelist.csv
 ```
 
-### Access to OCI registries (Optional)
+#### Usage arguments
 
+| Argument type | Options                                | Required |
+| ------------- | -------------------------------------- | -------- |
+| -profile      | staphylococcus_aureus/escherichia_coli | True     |
```
Comment (svartapaerlan) [inline README.md:38]: `mycobacterium_tuberculosis` and `klebsiella_pneumoniae` also work for profiles

File: configs/nextflow.ngp.config [hunk 2]
```diff
@@ -43,6 +52,7 @@ profiles {
 		params.species = "staphylococcus aureus"
 		params.speciesDir = 'saureus'
 		params.mlstScheme = 'saureus'
+		params.symlinkDir = "/access/jasen/saureus/"
```
Comment (svartapaerlan) [inline configs/nextflow.ngp.config:55]: Does it not work if `params.symlinkDir` is not provided?

File: container/Makefile [hunk 3]
```diff
@@ -104,14 +104,11 @@ $(remote_containers):
 build_local_containers: $(local_containers)
 
 $(local_containers):
-ifneq ($(shell id -u), 0)
```
Comment (svartapaerlan) [inline container/Makefile:107]: This is to test if the user has sudo priviedges no?

Comment (sylvinite) [reply container/Makefile:107]: no longer a req w a recent apptainer

File: configs/nextflow.ngp.config (hunk 2 above)
Comment (sylvinite) [reply configs/nextflow.ngp.config:55]: Yea it was a 2am solution to suppress the warnings. Same w the devmode flag. Good catch

**Review by ryanjameskennedy — APPROVED** [review]
LGTM!

**Review by svartapaerlan — APPROVED** [review]
LGTM!

**Pull request merged**